dvc/repo/__init__.py

```diff
@@ -165,6 +166,7 @@ def __init__(
 
         self.cache = Cache(self)
         self.cloud = DataCloud(self)
+        self.stage = StageLoad(self)
```

Comment:
Suggestion for better naming would be appreciated.

Reply:
@pared, will make it `StageLoader`, and change the other one to `MultiStageLoader`. 

> Maybe stage_load or stage_resolver will be ok, depending on what name we choose in the end.

`stage_load,load_one()` looks a bit odd to me. :) But, I understand your concern.